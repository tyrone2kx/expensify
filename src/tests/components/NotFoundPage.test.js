import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/ExpenseListItem';


test('should return not found page', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
});

