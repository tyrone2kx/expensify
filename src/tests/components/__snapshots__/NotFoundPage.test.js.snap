// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return not found page 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/undefined"
  >
    <h3 />
    <p>
      $0.00
       -
      December 31st, 1969
    </p>
  </Link>
</div>
`;
